### ISSUE TITLE
Include version in top of CSS file again?

### ISSUE THREAD

**Issue by henrik** [FortAwesome/Font-Awesome#14518]
Hi! Recently upgraded from 5.0.13 to 5.6.3. Not using any kind of dependency manager for Font Awesome, just copying files into the project.

5.0.13 included the version number in a comment at the top of all.css, and 5.6.3 didn't.

I liked how I could easily look up the version in the file to determine which icons are available to me, and I liked that the diff clearly showed the old and new version when upgrading.

I can add it in manually, of course, but could there be a point to putting it back in the distributed files also?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

> 5.0.13 included the version number in a comment at the top of all.css, and 5.6.3 didn't.

I can see the license information on each one of:
- all.css https://github.com/FortAwesome/Font-Awesome/blob/master/css/all.css#L2
- fontawesome.css https://github.com/FortAwesome/Font-Awesome/blob/master/css/fontawesome.css
- fontawseom.min.css https://github.com/FortAwesome/Font-Awesome/blob/master/css/fontawesome.min.css

but the downloadable zip bundle for version 5.6.3 does not contain it.

The license header is present in the 5.6.1 bundle, so I think there is a regression somewhere in the build system

@robmadole could you please take a look ?

**Comment by tagliala** [comment]
Well, I'm going to close here but feel free to comment

**Comment by henrik** [comment]
Thank you! I tried downloading it again (https://origin.fontawesome.com/releases/5.7.0/web/download when logged in) and it seems it's not at the top of `all.css`:

```
$ head -n 5 fontawesome-pro-5.7.0-web/css/all.css
.fa,
.fas,
.far,
.fal,
.fab {
```

**Comment by tagliala** [comment]
@henrik I can confirm, but this is related to Font Awesome Pro now

Could you please open this issue @ https://github.com/FortAwesome/Font-Awesome-Pro ?

**Comment by tagliala** [comment]
@henrik if you are a pro user you should have access to that repo

Please check that you have linked your Font Awesome Account to your GitHub account at https://fontawesome.com/account/services

**Comment by tagliala** [comment]
Reopening here, this issue came back on 5.7.1 and it also affects JS files

**Comment by robmadole** [comment]
Ok, I think I see what the issue was. This should be fixed in 5.7.2. We'll see 🤞

**Comment by tagliala** [comment]
Still an issue in 5.7.2 (Pro-Free-CDN / CSS-JS) :\

**Comment by robmadole** [comment]
$%^&*! Ok, @tagliala can you give me a link to a file that _should_ have the header and doesn't? (Sorry for being dense here)

**Comment by tagliala** [comment]
@robmadole 

- css and js files in the downloadable zip bundles (both font awesome pro and free)
- https://github.com/FortAwesome/Font-Awesome/releases/download/5.7.2/fontawesome-free-5.7.2-web.zip ❌ (same for FA Pro)

CDN:
- https://use.fontawesome.com/releases/v5.6.1/css/all.css ✔️ 
- https://use.fontawesome.com/releases/v5.6.1/js/all.js ✔️ 
- https://use.fontawesome.com/releases/v5.6.3/css/all.css ❌ (and newer)
- https://use.fontawesome.com/releases/v5.6.3/js/all.js ❌ (and newer)

**Comment by robmadole** [comment]
Thanks @tagliala that helps. We aren't going to be able to change the older versions but I'll try to get this really fixed before the next version is released.

**Comment by robmadole** [comment]
OK, I've verified that this is at least fixed in 5.8.0. We aren't going to be able to fix the older ones but I think I've fixed the root cause that was responsible for this.
